src/ontology/mondo-edit.obo

```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7100" xsd:anyURI
 
+[Term]
+id: MONDO:0100539
+name: hemiplegic migraine-developmental and epileptic encephalopathy spectrum
+def: "A spectrum in which individuals may present with phenotypes ranging from hemiplegic migraines without epilepsy to developmental and epileptic encephalopathy with or without episodic hemiplegia or other forms of paresis. Symptoms and severity may vary within families." [https://clinicalgenome.org/affiliation/40005/]
+is_a: MONDO:0018925 {source="https://clinicalgenome.org/affiliation/40005/"} ! familial or sporadic hemiplegic migraine
+is_a: MONDO:0100062 {source="https://clinicalgenome.org/affiliation/40005/"} ! developmental and epileptic encephalopathy
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7195" xsd:anyURI
+
+[Term]
+id: MONDO:0100540
+name: GATA6-related congenital heart disease with or without pancreatic agenesis or neonatal diabetes
```

Comment:
See comment in ticket: https://github.com/monarch-initiative/mondo/issues/7218#issuecomment-1933116976